cluster/addons/metadata-agent/stackdriver/metadata-agent.yaml

```diff
@@ -58,6 +58,11 @@ spec:
       restartPolicy: Always
       schedulerName: default-scheduler
       terminationGracePeriodSeconds: 30
+      tolerations:
```

Comment:
Discussion topic: do we need similar tolerations in the cluster-level agent spec? I suspect the only reason we would is if all nodes are tainted. @x13n @MaciekPytel @kawych @piosz @bmoyles0117

Reply:
Some of that conversation happened in https://github.com/kubernetes/kubernetes/pull/68920#discussion_r221987620.

Reply:
Ah, thanks for the pointer. I see the priority class now. Given that we'll only cherry-pick this into 1.12 and 1.11, this should work. If we ever need a 1.10 cherry-pick, we'd have to add stronger tolerations (e.g., `CriticalAddonsOnly`).

Reply:
`CriticalAddonsOnly` isn't really stronger - the ones added here are wildcard ones, so they include it already.

Reply:
I meant for the cluster-level agent, which currently doesn't have any tolerations.

Reply:
Oh, right. I think if someone is tainting all their nodes, something may break anyway, but yes, this could at least mitigate the problem in 1.10.

Reply:
I think a lot of things won't schedule if you taint all your nodes (DNS, metrics-server, ...). I don't think cluster-level agent should have stronger tolerations than other cluster-level system components.